packages/cerebral-provider-http/src/utils.js

```diff
@@ -64,3 +64,20 @@ export function convertObjectWithTemplates (obj, context) {
     return convertedObject
   }, {})
 }
+
+export function parseHeaders (rawHeaders) {
```

Comment:
Is it necessary to parse the headers? Can you not just grab them? Not quite sure if HTTP should be responsible for it :)